int64 id
int64 t
int64 gaze
float64 hand_x
float64 hand_y
int64 grip
bool target_reached  
